src/Content/Backend/Solution/Monaco.Template.Backend.Common.BlobStorage.Tests/Monaco.Template.Backend.Common.BlobStorage.Tests.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Please make sure to include this project folder in the corresponding exclude list on the `template.json` file in order to prevent including the whole project on the generated solution when the `filesSupport` flag is not enabled.